pkg/clients/edgeconnect/edgeconnect.go

```diff
@@ -28,12 +28,20 @@ type GetResponse struct {
 	Metadata         Metadata         `json:"metadata"`
 }
 
+type GetEdgeConnectsResponse struct {
```

Comment:
```suggestion
type ListResponse struct {
```
(related to my other comment)